README.md

```diff
@@ -213,26 +278,26 @@ As already discussed, employees can get restricted stock, stock options, or RSUs
     - 🔹 Founders and very early employees will almost always want to do an 83(b) election upon the receipt of unvested shares, since the stock value is probably low. If the value is really low, and the taxes owed are not that great, you can make the election without having to pay much tax and start your capital gains holding period on the shares.
 
 ### 409A valuations
 
 - When stock vests, or you exercise an option, the IRS will consider what the **fair market value (FMV)** of the stock is when determining the tax you owe.
 - Of course, if no one is buying and selling stock, as is the case in most startups, then its value isn’t obvious. For the IRS to evaluate how much stock is worth, it uses what is known as the [**409A valuation**](http://www.fenwick.com/FenwickDocuments/409_Valuations_Stock_Options.pdf) of the company.
-    - The startup pays for an appraisal that sets the 409A, typically annually or after events like fundraising. In practice, this number could be low or high.
+    - The startup pays for an appraisal that sets the 409A, typically annually or after events like fundraising.
     - 🔹 A company wants the 409A to be low, so that employees make more off options, but not low enough the IRS won’t consider it reasonable. Typically, the 409A is much less than what investors pay for preferred stock; often, it might be only a third of the preferred stock price.
 
 ### Stock awards vs ISOs vs NSOs
 
 - Startups generally decide to give ISOs or NSOs depending on the legal advice they get. It’s rarely up to you which you get, so you need to know about both. There are pros and cons of each from both the recipient’s and the company’s perspective.
-    - 🔸 ISOs cannot be granted to non-employees (i.e., independent contractors).  
+    - 🔸 ISOs cannot be granted to non-employees (like independent contractors).  
 - ❗🔹 **When you owe tax**: When you get stock options and are considering if and when to exercise them, you need to think about the taxes. In principle, you need to think about taxes (1) at time of grant; (2) at time of exercise; and (3) at time of sale.
 - These events trigger ordinary tax (high), long-term capital gains (low), or AMT (possibly high) taxes in different ways for NSOs and ISOs. The taxes will depend on the gain (sometimes called spread) between the strike price and the FMV, known as the [**bargain element**](http://www.investorwords.com/5414/bargain_element.html), and the gain on the sale. This isn’t the whole story, but from an employee’s point of view, the key differences are (see [here](https://www.irs.gov/taxtopics/tc427.html), [here](http://www.startuplawblog.com/2013/05/15/incentive-stock-options-vs-nonqualified-stock-options/), [here](http://www.investopedia.com/articles/optioninvestor/07/esoabout.asp), and [here](http://www.startuplawblog.com/2010/08/11/top-reasons-nqos-over-isos/)):
```

Comment:
I have literally never heard someone call this the bargain element, but I often hear it called the spread.

Reply:
✔️